code/modules/farming/soil.dm

```diff
@@ -885,6 +890,12 @@
 	if(plant_health <= MAX_PLANT_HEALTH * 0.3)
 		growth_multiplier *= 0.75
 
+	// Limit non-mushrooms from mushroom mounds and apply boosts to mushrooms
+	if(mushmound && !plant.mound_growth)
+		return
+	if(mushmound && plant.mound_growth)
+		growth_multiplier *= 1.2
+		nutriment_eat_multiplier *= 0.8
```

Comment:
```suggestion
	if(istype(src, /obj/stucture/soil/mushmound))
		growth_multiplier *= 1.2
		nutriment_eat_multiplier *= 0.8
```
This sucks but oh well. what you had before would prevent regular plants from working properly